### PR TITLE
SONAR-6152 Customize look&feel with new logo

### PR DESCRIPTION
**Pull request by simonbrandhof**
SONAR-6152 Customize look&feel with new logo

### CODE REVIEW

**Comment by simonbrandhof** [comment]
Stas, up to you to use the new parameters sonar.lf.logoUrl and sonar.lf.logoWidthPx. Thanks

File: sonar-core/src/main/java/org/sonar/core/config/CorePropertyDefinitions.java [hunk 1]
```diff
@@ -43,6 +43,18 @@ private CorePropertyDefinitions() {
     defs.addAll(DataCleanerProperties.all());
 
     defs.addAll(ImmutableList.of(
+      // WEB LOOK&FEEL
+      PropertyDefinition.builder("sonar.lf.logoUrl")
+        .deprecatedKey("sonar.branding.image")
+        .name("Logo URL")
+        .description("URL to logo image. Any standard format is accepted.")
+        .build(),
+      PropertyDefinition.builder("sonar.lf.logoWidthPx")
```
Comment (stas-vilchik) [inline sonar-core/src/main/java/org/sonar/core/config/CorePropertyDefinitions.java]: we can drop this property, so the image will be fluid

Comment (stas-vilchik) [inline sonar-core/src/main/java/org/sonar/core/config/CorePropertyDefinitions.java]: Please also add sonar.lf.logoTitle for the tooltip

**Comment by simonbrandhof** [comment]
@jblievremont I was inspired by jira.lf.* properties (see https://confluence.atlassian.com/display/JIRA/Customizing+the+Look+and+Feel)

**Comment by simonbrandhof** [comment]
@stas-vilchik a 10px padding is missing on the left of icon, so it's vertically aligned with content.

**Comment by simonbrandhof** [comment]
LGTM

**Pull request merged**